src/java.base/share/classes/sun/security/util/KnownOIDs.java

```diff
@@ -144,6 +144,7 @@
     SHA3_512("2.16.840.1.101.3.4.2.10", "SHA3-512"),
     SHAKE128("2.16.840.1.101.3.4.2.11"),
     SHAKE256("2.16.840.1.101.3.4.2.12"),
+    SHAKE256_LEN("2.16.840.1.101.3.4.2.18", "SHAKE256-LEN"),
```

Comment:
Can we move this down a little? The ordering within the section is based on the oid value. It's easier to check for unlisted/unsupported oid value this way.

Reply:
I'll add one.